ext/standard/tests/file/bug28790.cache.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+Bug #28790: Add php.ini option to disable stat cache (with cache)
+--CREDITS--
+jnoll <jnoll [at] prim [dot] hu>
+Kevin Lyda <kevin [at] lyda [dot] ie>
```

Comment:
CREDITS can be dropped as handled by git now (you can use Co-Authored-By for multiple authors).

Reply:
Done.